datahub-graphql-core/src/main/resources/entity.graphql

```diff
@@ -3209,6 +3265,237 @@ input InstitutionalMemoryMetadataUpdate {
     createdAt: Long
 }
 
+"""
+A Notebook Metadata Entity
+"""
+type Notebook implements Entity {
+    """
+    The primary key of the Notebook
+    """
+    urn: String!
+
+    """
+    A standard Entity Type
+    """
+    type: EntityType!
+
+    """
+    The Notebook tool name
+    """
+    tool: String!
```

Comment:
Need a platform field to be retrieved from the DataPlatformInstance aspect
```    
"""
Standardized platform.
"""
platform: DataPlatform!
```

Reply:
yeah, you are correct. This is a direct copy from our private repo which has not included dataPlatformInstance aspect yet

Reply:
I am a little confused. I noticed that there is also an entity called "DataPlatformInstance". Since the aspect is "
"dataPlatformInstance", should we use "DataPlatformInstance" instead of "DataPlatform"?

Reply:
Ideally we should. This was an aspect added later on, so we don't have a type for it as of now, which was why I recommended above. Feel free to add DataPlatformInstance type in this PR and attach it.

Reply:
Thanks for the context. I will use DataPlatform.